src/main/java/uk/gov/digital/ho/hocs/search/application/queue/DataChangeType.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.digital.ho.hocs.search.application.queue;
+
+public enum DataChangeType {
+    CASE_CREATED("CASE_CREATED"),
+    CASE_UPDATED("CASE_UPDATED"),
+    CASE_DELETED("CASE_DELETED"),
+    CASE_COMPLETED("CASE_COMPLETED"),
+    CORRESPONDENT_CREATED("CORRESPONDENT_CREATED"),
+    CORRESPONDENT_DELETED("CORRESPONDENT_DELETED"),
+    CORRESPONDENT_UPDATED("CORRESPONDENT_UPDATED"),
+    CASE_TOPIC_CREATED("CASE_TOPIC_CREATED"),
```

Comment:
Just me being pedantic, could we alphabetise please?

Reply:
not quite, I've organised them to case_ then case_topic but case_topic would come between case_deleted and case_updated